funnel/models/project.py

```diff
@@ -485,6 +488,17 @@ def roles_for(self, actor=None, anchors=()):
         roles.update(self.profile.roles_for(actor, anchors))
         return roles
 
+    @with_roles(call={'admin'})
+    def assign_labelset(self, labelset):
```

Comment:
Where is this method needed? Labelsets never get reassigned. They are permanently a part of the project they were created in.